### DISCUSSION TITLE
Ability to set a custom commit message when applying a suggestion

### DISCUSSION THREAD

**Discussion by nre-ableton** [community/community#12201] [Mobile]
Our company relies very heavily on fixup commits, which we squash (using `git rebase --autosquash`) before merging. As such, it is important that when applying suggestions, we can give them sensible commit messages like `fixup! <title of commit to fixup>` or `squash! <title of commit to squash>`. This is possible on the web but not in the mobile client.

**Comment by stevepopovich** [comment]
Hi Nik, thanks for the feedback. This would be a good addition to our app. Passing this request on to the team for prioritization. Thanks!

**Reply by nre-ableton** [reply]
Great, thanks for considering it! My team would really appreciate it.
